### PR TITLE
i18n(es): Update `nav.ts`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

* [`c6bdb30`](https://github.com/withastro/docs/commit/c6bdb308baf83178fd3c9f412546d988febd3b9a)
* [`ce35815`](https://github.com/withastro/docs/commit/ce358157988493093d411b2f2b1b1c1dd7126d3b)
* [`30f6033`](https://github.com/withastro/docs/commit/30f603347ef8baabae85a05f396a8bb6cf342e74#diff-154f7ce87358ad1a8702c608db21ac7f1dc44025ded816eefdb9c1e0bda88961)

### CODE REVIEW

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo, te dejo una sugerencia @Waxer59 😁

File: src/i18n/es/nav.ts [hunk 1]
```diff
@@ -51,6 +52,8 @@ export default NavDictionary({
 	'guides/data-fetching': 'Fetching de Datos',
 	'guides/middleware': 'Middleware',
 	'guides/testing': 'Testing',
+	'guides/prefetch': 'Precargar',
+	'guides/internationalization': 'internacionalización',
```
Comment (dreyfus92) [inline src/i18n/es/nav.ts:56]:
```suggestion
	'guides/internationalization': 'Internacionalización',
```

**Comment by Waxer59** [comment]
Muchas gracias por la revisión 🤠!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**